### PR TITLE
2.4.5 fix for #93

### PR DESCRIPTION
**Pull request by meisnate12**
### Fixed
- #93 - actually fixed max_age

### CODE REVIEW

File: app/plex_tools.py [hunk 1]
```diff
@@ -232,11 +232,17 @@ def search_plex():
                     modifier = sf[0][-4:]
                     method = subfilter_alias[sf[0][:-4]] if modifier in [".not", ".lte", ".gte"] else subfilter_alias[sf[0]]
                     if method == "max_age":
-                        threshold_date = datetime.now() - timedelta(days=sf[1])
-                        attr = getattr(current_m, "originallyAvailableAt")
-                        if attr < threshold_date:
-                            match = False
-                            break
+                        try:
+                            max_age = regex_first_int(sf[1], "max_age")
```
Comment (mza921) [inline app/plex_tools.py:236]:
```
| Updating collection: Test...
| Sync Mode: sync
|
|  Collection Filter max_age: 30
|
|
| Processing all:
Traceback (most recent call last):
  File ".\plex_auto_collections.py", line 819, in <module>
    update_from_config(config_path, plex, True, args.no_meta, args.no_images)
  File ".\plex_auto_collections.py", line 450, in update_from_config
    missing, map = add_to_collection(config_path, plex, m, v, c, map, filters)
  File "app\plex_tools.py", line 307, in add_to_collection
    max_age = regex_first_int(sf[1], "max_age")
  File "app\plex_tools.py", line 23, in regex_first_int
    id = re.search('(\\d+)', str(data)).group(1)
NameError: name 're' is not defined
```

Should be a simple fix to add `re`, but I just want to remove the `y` option altogether.

Comment (meisnate12) [reply app/plex_tools.py:236]: ok its probably less confusing anyways

Comment (mza921) [reply app/plex_tools.py]: It looks like it's working now, but this is so slow on large libraries, I almost want to warn people.

Comment (meisnate12) [reply app/plex_tools.py]: yea it has to check every single movie its alot tbh but i can't think of another way to do it. really any collection that you build just using filters is going to be very slow

**Pull request merged**